recipes/toad/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "toad" %}
+{% set version = "0.0.58" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 9dce218ac11eb9e3a80b233557694d0d349418a1ba517306919dff02595e5f07
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
```

Comment:
Looks like `toad requires Python >=3.5, so please use `python >=3.5` in both `host` and `run` sections.

Reply:
Thanks, could you help me to pass the check? It's still failed after I fixed the python version.